### ISSUE TITLE
Doesn't rename tags in svelte files

### ISSUE THREAD

**Issue by SebasF1349** [windwp/nvim-ts-autotag#164]
The plugin does auto close html tags, but doesn't rename them in svelte files (it works on html files). This is one of the many configs I tried.

``` lua
{
    "windwp/nvim-ts-autotag",
    ft = { "html", "svelte", "markdown" },
    dependencies = "nvim-treesitter/nvim-treesitter",
    config = function()
      require("nvim-ts-autotag").setup({
        enable = true,
        enable_rename = true,
        enable_close = true,
        enable_close_on_clash = true,
        filetypes = { "svelte", "html" },
      })
    end,
}
```

The svelte parser is installed. I also tried adding the plugin in the treesitter config with this but with the same effect:
```lua
require'nvim-treesitter.configs'.setup {
  autotag = {
    enable = true,
    enable_rename = true,
    enable_close = true,
    enable_close_on_slash = true,
    filetypes = { "html" , "xml" },
  }
}
```
I checked many issues here but no solution worked for me. I'm not using any nvim distro. Any suggestion?

**Comment by harrisonn26** [comment]
I'm having the same issue. Renaming works fine in html and tsx files, but not svelte files. Auto closing is working fine.

**Comment by roycrippen4** [comment]
+1 Same problem as you @harrisonn26. Works great for html and jsx, but only getting auto-closing for svelte files.

**Comment by roycrippen4** [comment]
I just made a PR for this issue @harrisonn26 and @SebasF1349. It's a super simple one line change if you want to go change it in the meantime.

**Comment by RubyHuntsman** [comment]
Link to @roycrippen4 pr  https://github.com/windwp/nvim-ts-autotag/pull/167

**Comment by SebasF1349** [comment]
Sorry that I forgot to reply, but it works.

**Comment by PriceHiller** [comment]
Should be closed by #173, as that PR was integrated in that fork which is now in the process of being merged back in.

**Comment by PriceHiller** [comment]
#173 was merged. This *should* be resolved now.

I'll be closing this issue -- please ping me if this is still a problem and I'll reopen and investigate.

Thanks 🙂
